<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shop</class>
 <widget class="QWidget" name="Shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>å•†åº—</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- å…¨å±€èƒŒæ™¯ï¼Œç±»ä¼¼æš‚åœèœå• --- */
QWidget#Shop {
    background-color: rgba(0, 0, 0, 200); /* å¢åŠ ä¸é€æ˜åº¦ */
}

/* --- æ ‡é¢˜ --- */
QLabel#titleLabel {
    font-size: 52px;
    font-weight: bold;
    color: #FFD700;
    padding: 15px;
}

/* --- åˆ·æ–°å’Œé‡‘å¸åŒºåŸŸ --- */
#topBarContainer {
    background-color: rgba(255, 255, 255, 0.05);
    border-radius: 10px;
    padding: 5px;
}
QLabel#coinLabel {
    font-size: 24px;
    font-weight: bold;
    color: white;
    padding: 8px 15px;
}

/* --- å•†å“å¡ç‰‡ --- */
QWidget#itemCard {
    background-color: rgba(30, 30, 30, 230); /* ä½¿ç”¨æš‚åœèœå•çš„é¢æ¿é¢œè‰² */
    border: 1px solid rgba(80, 80, 80, 200);
    border-radius: 15px;
    padding: 20px;
}

/* --- å­—ä½“å¢å¤§ --- */
QLabel#itemName {
    font-size: 22px; /* å¢å¤§ */
    font-weight: bold;
    color: white;
}
QLabel#itemDesc {
    font-size: 16px; /* å¢å¤§ */
    color: #B0B0B0;
}

/* --- æŒ‰é’®æ ·å¼ --- */
QPushButton {
    font-size: 18px; /* å¢å¤§ */
    font-weight: bold;
    color: white;
    background-color: #28a745;
    border: none;
    border-radius: 8px;
    padding: 12px;
    margin-top: 10px;
}
QPushButton:hover {
    background-color: #218838;
}
QPushButton:disabled {
    background-color: #5A5A5A;
    color: #999;
}
QPushButton#refreshButton {
    background-color: #17a2b8; /* åˆ·æ–°æŒ‰é’®ç”¨é’è‰² */
    padding: 8px 15px;
}
QPushButton#refreshButton:hover {
    background-color: #138496;
}
QPushButton#continueButton, QPushButton#backToMenuButton {
    background-color: #007bff;
    min-height: 40px;
}
QPushButton#continueButton:hover, QPushButton#backToMenuButton:hover {
    background-color: #0069d9;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topBarLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>è¡¥ç»™å•†åº—</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="topBarContainer" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="toolTip">
           <string>èŠ±è´¹10é‡‘å¸åˆ·æ–°å•†å“</string>
          </property>
          <property name="text">
           <string>ğŸ”„ åˆ·æ–° (10)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="coinLabel">
          <property name="text">
           <string>ğŸ’° 0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="itemsGridContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>25</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="backToMenuButton">
       <property name="text">
        <string>è¿”å›ä¸»èœå•</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ç»§ç»­ä¸‹ä¸€æ³¢</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
